aws-bill-pdf-to-csv/README.md

```diff
@@ -0,0 +1,50 @@
+This sample takes in a S3 URI pointing at a AWS Bill PDF and converts the bill into a CSV. Each item in the bill is converted to a row in the CSV. It will then write the CSV back to a provided S3 URI. This showcases how Griptape can be used to automate previously manual processes such as identifying if a PDF field with identical formatting is a `AWS Region` or an `AWS Service`.
+
+[![Deploy_to_Griptape](https://github.com/griptape-ai/griptape-cloud/assets/2302515/4fd57873-5c93-44a8-8fa3-ac1bf7d73bcc)](https://cloud.griptape.ai/structures/create?sample-name=griptape-csv-filter&type=sample)
```

Comment:
update the sample-name to the name of the folder

Reply:
Added (and renamed)